src/pages/home.tsx

```diff
@@ -1,15 +1,25 @@
 import { Box, Flex } from '@chakra-ui/react'
 import { SkeletonPostList } from '~/components/SkeletonPostList'
 import Suspense from '~/components/Suspense'
-import { APP_GRID_COLUMN, APP_GRID_TEMPLATE_COLUMN } from '~/constants/layouts'
+import {
+  ADMIN_NAVBAR_HEIGHT,
+  APP_GRID_COLUMN,
+  APP_GRID_TEMPLATE_COLUMN,
+} from '~/constants/layouts'
 import { NewPostBanner, PostList } from '~/features/home/components'
 import { type NextPageWithLayout } from '~/lib/types'
 import { AppGrid } from '~/templates/AppGrid'
 import { AdminLayout } from '~/templates/layouts/AdminLayout'
 
 const Home: NextPageWithLayout = () => {
   return (
-    <Flex w="100%" flexDir="column">
+    <Flex
+      w="100%"
+      flexDir="column"
+      position={{ base: 'absolute', sm: 'inherit' }}
```

Comment:
comment: the whole page being absolute may cause layout problems in the future that is hard to debug. but approving for now